keyboards/handwired/riblee_f401/keymaps/riblee/keymap.c

```diff
@@ -0,0 +1,353 @@
+/* Copyright 2015-2017 Jack Humbert
+ * Copyright 2020 Daniel Reibl <janos.daniel.reibl@protonmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum preonic_layers {
+    _QWERTY,
+    _COLEMAK,
+    _DVORAK,
+    _LOWER,
+    _RAISE,
+    _ADJUST
+};
+
+enum preonic_keycodes {
+    QWERTY = SAFE_RANGE,
+    COLEMAK,
+    DVORAK,
+    BACKLIT,
+    MAC,
+    LINUX
+};
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
+
+// Tap Dance declarations
+enum {
+    TD_A,
+    TD_E,
+    TD_I,
+    TD_O,
+    TD_U,
+};
+
+const uint8_t shift = MOD_BIT(KC_LSFT) | MOD_BIT(KC_RSFT);
+
+// Tap Dance functions
+void dance_key_a (qk_tap_dance_state_t *state, void *user_data) {
+    if (state->count == 1) {
+        SEND_STRING("a");
+        reset_tap_dance(state);
+    } else if (state->count == 2) {
+        if (!(keyboard_report->mods & shift)) {
+            send_unicode_hex_string("00E1"); // á
```

Comment:
You can do this instead:
```suggestion
            send_unicode_string("á");
```

Reply:
Thanks, it's fixed now for all accent keys